### DISCUSSION TITLE
Open Shell Not Listing All Apps

### DISCUSSION THREAD

**Discussion by Philn457** [Open-Shell/Open-Shell-Menu#1859] [Q&A]
The open shell Apps list only has about half the apps that are loaded on my Windows 11 computer. 
An example is Zoom.  This app displays in the Windows app list but not in the Open Shell list.  
Any suggestions on how I can get the complete list?   
Thanks,

**Comment by ge0rdi** [comment]
Could it be that Modern apps are missing?

Do you have `Show Apps menu` enabled?
There should be `Apps` folder (directly below `Programs` or inside `All Programs` in case of Win7 style).

**Reply by Philn457** [reply]
Show Apps folder is enabled and is listed below the Programs folder.   
It does appear that more "Modern" apps are missing from the list. 
I have been through all the Open Shell settings, changed a few, but no change to the Apps list being incomplete.

**Reply by ge0rdi** [reply]
You can try to obtain `Open-Shell` log using this [guide](https://github.com/Open-Shell/Open-Shell-Menu/wiki/How-to-obtain-advanced-Open-Shell-Menu-log).
Maybe there will be some hint about those missing apps.

**Comment by Philn457** [comment]
A quick check shows that most of the missing apps show up as Programs in the Open Shell menu.   
It looks to me that how Microsoft defines "Apps" and how Open Shell defines them are two different things. 
This issue will remain until the Open Shell program defines apps the same way that Microsoft does. 
Not a big issue if you know it exists.
